### PR TITLE
fixed:#246

### LINKED ISSUE #246: Showing error in logging using bootstrap alert instead of alert.
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [ x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/susi_server

**Current behavior:**
If there is any error in logging then it is shown using alert.

**Expected behavior:**
The error will be shown using bootstrap alert class in the login page itself
screenshot 2

**Steps to reproduce:**

**Related code:**

```
insert any relevant code here else remove this section
```

**Other information:**

**System information:**
Windows 10

```

Your operating system
```
Windows 10
**Would you like to work on the issue?**

Please let us know if you can work on it or the issue should be assigned to someone else.
```
Yes I can work on it

### PR DESCRIPTION
**Pull request by saikat041**
Fixes #246

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
Now the error in logging in is shown using bootstrap alert method.

#### Changes proposed in this pull request:
screenshot 19

-
-
-

### CODE REVIEW

**Comment by ms10398** [comment]
Please squash commits

File: src/js/login.js [hunk 1]
```diff
@@ -51,6 +60,7 @@ loginForm.addEventListener("submit", function login(event){
 		return;
 	}
 	$("#loginbutton").button("loading");
+  errorBlock.style.visibility="hidden";
```
Comment (gabru-md) [inline src/js/login.js]: Indent properly

File: src/js/login.js [hunk 2]
```diff
@@ -5,23 +5,32 @@ var logoutButton = document.getElementById("logout");
...
+            showLoggedInBlock(false);
+            }
+    });
+
+
 
 };
 
+function showError(errorMsg) {
+  errorBlock.style.visibility="visible";
```
Comment (gabru-md) [inline src/js/login.js]: Indent here as well.

**Comment by saikat041** [comment]
@ms10398 please review it

**Comment by saikat041** [comment]
@ms10398 now its done

**Comment by saikat041** [comment]
@ms10398 please review it

**Comment by ms10398** [comment]
@gabru-md @stealthanthrax @sachincool Please review

**Comment by ms10398** [comment]
Yeah you can add it too and please reference `issue no.` in your pull request.

**Comment by saikat041** [comment]
@gabru-md But after successful log in the page is changed. And showing "You are logged in " in that page is unnecessary i think.

**Comment by stealthanthrax** [comment]
@saikat041 , you can add "you are logged in as: email" for the page

**Comment by gabru-md** [comment]
@saikat041 i get your point.
So I guess we better change the alert text from `alert(response.message)` to something more relevant.
Can you please do it?
something like : `alert("Logged in!")` ?

**Comment by gabru-md** [comment]
I guess I did not see that! It already says `Logged in as : email`.
@ms10398 can you confirm if I am right ?

**Comment by ms10398** [comment]
Yes its there @gabru-md

**Comment by saikat041** [comment]
screenshot 21

Can we use this instead of alert("You are logged in as  email")

**Comment by gabru-md** [comment]
Sure @saikat041 
Please commit it and squash your commits so that it can be merged!

**Comment by saikat041** [comment]
screenshot 23
Now all error are shown using bootstrap alert method.

**Pull request merged**